dubbo-rpc/dubbo-rpc-rest/src/main/java/org/apache/dubbo/rpc/protocol/rest/RestProtocol.java

```diff
@@ -91,12 +88,11 @@ public int getDefaultPort() {
     protected <T> Runnable doExport(T impl, Class<T> type, URL url) throws RpcException {
         String addr = getAddr(url);
         Class implClass = ApplicationModel.getProviderModel(url.getServiceKey()).getServiceInstance().getClass();
-        RestServer server = servers.get(addr);
-        if (server == null) {
-            server = serverFactory.createServer(url.getParameter(Constants.SERVER_KEY, DEFAULT_SERVER));
-            server.start(url);
-            servers.put(addr, server);
-        }
+        RestServer server = servers.computeIfAbsent(addr, addr0 -> {
```

Comment:
I also think we  should replace addr0 into restServer.   the addr0 is not clear for people  to understand.

Reply:
but I'm wondering why renaming a variable of type `String`  to `restServer` increases readability

Reply:
This has nothing to do with whether it is a `String` type, but using `restServer` in the meaning description will be clearer than `addr0`.
This is also to let others see the source code, know that the second parameter of the `servers` is a `RestServer` type.

Reply:
@scxwhite @CrazyHZM Since the changes is controversial, I've reverted the changes, as for 
> This is also to let others see the source code, know that the second parameter of the `servers` is a `RestServer` type.
I'd rather suggest renaming the `servers` to something like `serversKeyedByAddress` or anything else but what you said.

Reply:
That part of the code does not need to reverted, I think you modified is good, just change `addr0` to `restServer`

Reply:
Changes are made as requested